cli/cmd/profile.go

```diff
@@ -55,6 +57,10 @@ func (options *profileOptions) validate() error {
 		return errors.New("You must specify exactly one of --template or --open-api or --proto or --tap")
 	}
 
+	// service profile generation based on tap data requires access to k8s cluster
+	if options.ignoreCluster && options.tap != "" {
+		return errors.New("Service profile generation based on tap data require access check to k8s cluster. You can not use --ignore-cluster here")
```

Comment:
This is a bit of a nit, but it feels a little wordy. How about something like:

```suggestion
		return errors.New(""--ignore-cluster and --tap flags are mutually exclusive; SP generation based on tap data requires access-check to k8s cluster")
```

This isn't exactly shorter, but we use similar wording [here](https://github.com/linkerd/linkerd2/blob/cbcc305b78edb9fe65fc0edee99a5722a949287b/cli/cmd/stat.go#L741-L747) and still provide a little explanation about why.

Reply:
I felt the same while writing this :)